h/cli/commands/groups.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+import click
+
+
+@click.group()
+def groups():
+    """Manage groups."""
+
+
+@groups.command()
```

Comment:
If you change this to `@groups.command('add-publisher-group')` then we don't have to deal with ugly underscores when using the CLI :)